.mage/docs.go

```diff
@@ -34,6 +36,25 @@ func execHugo(args ...string) error {
 	return execGo("run", append([]string{"-tags", "extended", "github.com/gohugoio/hugo", "-s", "./doc"}, args...)...)
 }
 
+func downloadFile(targetpath string, url string) (err error) {
```

Comment:
```suggestion
func downloadFile(url, target string) (err error) {
```

from-to is more sensible order to me